chopsticks-sample/src/main/resources/dev/chopsticks/sample/app/dstreams/dstreams-sample-worker-app.conf

```diff
@@ -0,0 +1,23 @@
+akka {
+  coordinated-shutdown.run-by-jvm-shutdown-hook = on
+
+  http.server.preview.enable-http2 = on
+
+  log-dead-letters = off
+  loglevel = INFO
+
+  stream.materializer.debug.fuzzing-mode = off
+}
+
+iz-logging {
```

Comment:
Should already inherit from https://github.com/shopstic/chopsticks/blob/142eff5c6855e447749df7ed9884446b4b78b076/chopsticks-fp/src/main/resources/reference.conf#L13